packages/safe-ethers-lib/src/EthersAdapter.ts

```diff
@@ -71,19 +65,14 @@ class EthersAdapter implements EthAdapter {
   getSafeContract({
     safeVersion,
     chainId,
-    isL1SafeMasterCopy,
+    singletonDeployment,
     customContractAddress
-  }: GetSafeContractProps): GnosisSafeContractEthers {
+  }: GetContractProps): GnosisSafeContractEthers {
     let contractAddress: string | undefined
```

Comment:
You could just assign this immediately via a ternary and not need the `if ... else`.

Reply:
There's actually a couple of other occurances in this file where you could do it too.